### PR TITLE
Wrap up licensing subproject

### PR DESCRIPTION
**Pull request by dims**
The dependencies for kubernetes/kubernetes is in good shape, we can handle any items left as part of area/code-organization (sig-arch) if needed.

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

#### What type of PR is this:

/kind cleanup

### CODE REVIEW

**Comment by dims** [comment]
cc @justaugustus @swinslow @nikhita

**Review by hasheddan — APPROVED** [review]
/lgtm
/hold for those tagged

File: licensing/README.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# Licensing
```
Comment (nikhita) [inline licensing/README.md:1]: Should we move this doc to the sig-arch directory in k/community - https://github.com/kubernetes/community/tree/master/sig-architecture?

Comment (dims) [reply licensing/README.md:1]: probably better nearer to where we point folks to when they update a dependency?
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: licensing/README.md (hunk 1 above)
Comment (nikhita) [reply licensing/README.md:1]: > probably better nearer to where we point folks to when they update a dependency?

:+1:

Comment (nikhita) [reply licensing/README.md:1]: created https://github.com/kubernetes/community/pull/5400

**Pull request merged**